### PR TITLE
Settings: Remove unnessary error handling code

### PR DESCRIPTION
**Pull request by torkelo**
Convert `valueAsString` in the `setting` package to not handle panics from value conversion, as there shouldn't be any actual chance of this happening. I read the ini library code, and when you request a string it doesn't perform any conversion AFAICT.

### CODE REVIEW

File: pkg/setting/setting.go [hunk 1]
```diff
@@ -838,6 +797,14 @@ func (cfg *Cfg) Load(args *CommandLineArgs) error {
 	return nil
 }
 
+func readSectionValueString(section *ini.Section, keyName string, defaultValue string) string {
+	if !section.HasKey(keyName) {
+		return defaultValue
+	}
```
Comment (aknuds1) [inline pkg/setting/setting.go:803]: Doesn't `section.Key(keyName).MustString(defaultValue)` have the same effect as this?

**Review by marefr — APPROVED** [review]
LGTM, but will let Arve review as well since he did a big refactoring of this file recently.

**Review by aknuds1 — COMMENTED** [review]
I guess `valueAsString` will panic if the key doesn't exist, due to a nil pointer error, and convert the panic to an error? That does look weird indeed. I would expect `valueAsString` to instead convert the missing key to its default argument.

Why don't we instead of making a new function, fix `valueAsString`?

**Comment by torkelo** [comment]
> Why don't we instead of making a new function, fix valueAsString?

yea, that sounds better!

**Comment by aknuds1** [comment]
@torkelo Do you want to make the changes, or should I push them to this branch?

**Comment by torkelo** [comment]
@aknuds1 if you can take over this PR that be great  this was just a quick POC to get the conversation started

**Comment by aknuds1** [comment]
@torkelo Alright, will do!

**Review by aknuds1 — APPROVED** [review]

**Review by marefr — APPROVED** [review]
LGTM

**Pull request merged**